data/json/monsters/mammal.json

```diff
@@ -1143,6 +1154,7 @@
     "fear_triggers": [ "SOUND", "PLAYER_CLOSE", "FRIEND_ATTACKED" ],
     "placate_triggers": [ "PLAYER_WEAK" ],
     "death_function": [ "NORMAL" ],
+    "harvest": "animal_medium",
```

Comment:
Sheep has size "LARGE", why give it harvest animal_medium? Although honestly, sheep probably should be size medium

Also will need to remove the BONES/FAT flags like you've done similarly on line 422